types/unist/index.d.ts

```diff
@@ -1,40 +1,63 @@
 // Type definitions for Unist 2.0
-// Project: https://github.com/syntax-tree/unist
+// Project: https://github.com/syntax-tree/unist-ts
 // Definitions by: bizen241 <https://github.com/bizen241>
 //                 Jun Lu <https://github.com/lujun2>
+//                 Hernan Rajchert <https://github.com/hrajchert>
+//                 Titus Wormer <https://github.com/wooorm>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 3.0
 
+export as namespace Unist
+
+// Syntactic units in unist syntax trees are called nodes.
 export interface Node {
+    // The variant of a node.
     type: string;
+
+    // Information from the ecosystem.
     data?: Data;
+
+    // Location of a node in a source document.
+    // Must not be present if a node is generated.
     position?: Position;
 }
 
 export interface Data {
-    [key: string]: any;
+    [key: string]: unknown;
 }
 
+// Location of a node in a source file.
 export interface Position {
+    // Place of the first character of the parsed source region.
     start: Point;
+
+    // Place of the first character after the parsed source region.
     end: Point;
-    /** >= 1 */
-    indent?: number[];
+
+    // Start column at each index (plus start line) in the source region,
+    // for elements that span multiple lines.
+    indent: number[];
 }
 
+// One place in a source file.
 export interface Point {
-    /** >= 1 */
+    // Line in a source file (1-indexed integer).
     line: number;
-    /** >= 1 */
+
+    // Column in a source file (1-indexed integer).
     column: number;
-    /** >= 0 */
+
+    // Character in a source file (0-indexed integer).
     offset?: number;
 }
 
+// Nodes containing other nodes.
 export interface Parent extends Node {
+    // List representing the children of a node.
     children: Node[];
 }
 
+// Nodes containing a value.
 export interface Literal extends Node {
     value: any;
```

Comment:
Values of `Data` has the type `unknown`, but `Literal.value` has type `any`.
Is this intentional?
```suggestion
     value: unknown;
```

Reply:
you are right, my bad... will fix and push again once I solve the travis failure as well. Didn't know there was a mdast already :P 

https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/461519743?utm_source=github_status&utm_medium=notification

Reply:
I see that the mdast types are mantained by @lujun2... would you recommend to create a separate PR for mdast first and then redoing this one, or add both types at the same type in this PR?

Reply:
Also, I notice that inside https://github.com/syntax-tree/unist-ts/blob/master/index.d.ts, @wooorm used `export as namespace Unist`, but in this package you were using it as a module, the same in `mdast`. Would you like me to revert it to using module instead of namespace?

Reply:
> would you recommend to create a separate PR for mdast first and then redoing this one, or add both types at the same type in this PR?

I think there is no need to create a separate PR just to increase the required TypeScript version.

Reply:
I wrapped stuff in `Unist` because then it’s scoped (I believe). Sometimes unist/mdast use the same names (such as Parent, but there’s a difference between unist.Parent and mdast.Parent).
But y’all know more about TS, so please let me know if I was wrong to do that!

Reply:
They are scoped by the module. So you could do
```
import * as Unist from 'unist';
import * as MDast fromt 'mdast'
const node: Unist.Parent;
const node2: MDast.Parent;
```

or

```
import {Node} from 'unist';
const node: Node;
```

I'll make the changes and recommit